lib/rubygems/package.rb

```diff
@@ -385,7 +385,13 @@ def extract_tar_gz io, destination_dir, pattern = "*" # :nodoc:
           FileUtils.chmod entry.header.mode, destination
         end if entry.file?
 
-        File.symlink(entry.header.linkname, destination) if entry.symlink?
+        if entry.symlink?
+          begin 
+            File.symlink(entry.header.linkname, destination)
+          rescue NotImplementedError
+            # Not all Rubies support symlinks
```

Comment:
@segiddins if symlink is not implemented (all Ruby up to MRI 2.3 on windows) what would the alternative be?  I believe before this support was added the behavior was to ignore symlink entries.    Seem untenable at the level that a platform without symlinks can properly install a gem which requires them.  Perhaps an alternate to this could be to check symlink capability when gem starts up and to print out a warning once if the gem actually has a symlink in it?